wrapper/cpp/main.go

```diff
@@ -101,6 +172,60 @@ func KeyvaultKeyEncryptDataCV(serverName *C.char, groupName *C.char, keyvaultNam
 	return C.CString(*response.Result)
 }
 
+//export KeyvaultKeyEncryptDataV2
+func KeyvaultKeyEncryptDataV2(serverName *C.char, groupName *C.char, keyvaultName *C.char, keyName *C.char, input *C.char, algorithm *C.char, cv *C.char, timeoutInSeconds C.int, outputBuffer *C.char, outputBufferSize *C.ulonglong, errMessageBuffer *C.char, errMessageSize C.ulonglong) C.int {
+	if (serverName == nil || groupName == nil || keyvaultName == nil || keyName == nil || input == nil || algorithm == nil || cv == nil || outputBufferSize == nil) {
+		copyErrorMessage(C.CString("Invalid Argument"), errMessageBuffer, errMessageSize)
+		telemetry.EmitWrapperTelemetry("KeyvaultKeyEncryptDataV2", C.GoString(cv), "", "InvalidArgument", C.GoString(serverName))
+		return C.int(Win32ErrorBadArg)
+	}
+
+	keyClient, err := getKeyvaultKeyClient(C.GoString(serverName), C.GoString(cv))
+	// if errror occurs, return an empty string so that caller can tell between error and encrypted blob

```

Comment:
>// if errror occurs, return an empty string so that caller can tell between error and encrypted blob [](http://example.com/codeflow?start=1&length=100)

outdated comment?